### PR TITLE
Add AWS Config vs CloudQuery

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

File: docs/cq_vs_others/aws_config.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# CloudQuery vs AWS Config
+
+AWS Config is the native AWS asset inventory provided by AWS.
+
+**Key Differences:**
+- **Resource Types**: CloudQuery [supports](https://hub.cloudquery.io) more than 250 types of resources while AWS Config currently supports about 120 types of resources (tables). Being an open-source project you can easily add the missing resources without being blocked by a vendor.
```
Comment (bbernays) [inline docs/cq_vs_others/aws_config.md:6]: Shouldn't this link to the AWS Provider instead?

File: docs/cq_vs_others/aws_config.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- **Vanilla PostgreSQL**: CloudQuery uses vanilla PostgreSQL which makes it play nicely with the whole SQL eco-system and gives you the ability to re-use other tools like Grafan/BI. AWS Config is using a propietry subset of SQL and database and thus doesn't give you the ability to re-use other tools easily.
+- **Cloud Agnostic**: CloudQuery gives you the ability to assess, audit and monior [multi-cloud and SaaS infrastructure](https://hub.cloudquery.io).
+- **Pricing**: CloudQuery is open-source and thus you will pay only for the hosting of your PostgreSQL (you can use RDS, or any other managed version) and the compute for running CQ binary (you can use lambda). AWS Config charges both per item recorded and rule evaluated.
```
Comment (bbernays) [inline docs/cq_vs_others/aws_config.md:9]: Do we have docs on how to deploy CQ we could link to?

**Review by ukmadlz — APPROVED** [review]

File: docs/cq_vs_others/aws_config.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- **Limits**: See limits section below. AWS Config has a lot of soft and hard limit (that cannot be increased). CloudQuery doesn't impose any of those limits and is bound mainly by the PostgreSQL instance and compute.
+- **Policy Langauge**: CQ uses an HCL configuration with SQL as the query engine to define rules. AWS Config uses json/yaml and subset SQL as the query engine.
+
+
+## Limits Comparision
+
+This will only list hard limits as soft limits can be increased (though it is not documented what is their max limit). Some limits are quite serious for large accounts (especially around number of rules/policies).
```
Comment (bbernays) [inline docs/cq_vs_others/aws_config.md:16]: Link to AWS limits?

**Review by roneli — APPROVED** [review]
LGTM, would add links @bbernays suggested

File: docs/cq_vs_others/aws_config.md (hunk 2 above)
Comment (yevgenypats) [reply docs/cq_vs_others/aws_config.md]: added

**Pull request merged**